lcls-twincat-common-components/Library/Version/Global_Version.TcGVL

```diff
@@ -7,8 +7,8 @@
 // This function has been automatically generated from the project information.
 VAR_GLOBAL CONSTANT
     {attribute 'const_non_replaced'}
-    stLibVersion_lcls_twincat_common_components : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, nFlags := 1, sVersion := '0.0.0');
+    stLibVersion_lcls_twincat_common_components_nrw : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, nFlags := 0, sVersion := '0.0.0');
```

Comment:
need to change this back

Reply:
confused here, the project is actually already named lcls_twincat_common_components

Reply:
I think we just need to resolve this
If you can't find it in the twincat interface it's ok to just revert this whole file using git